orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/ExpressionTransform.java

```diff
@@ -123,6 +124,7 @@ private Object transformString(Object source,
             String value = e.getValue(evaluationContext, String.class);
             if (value == null) {
               value = String.format("${%s}", e.getExpressionString());
+              isPartiallyEvaluated = true;
```

Comment:
Why does this mean it's partially evaluated?

Reply:
If we're in this block, we were not able to resolve a SpEL expression, so just dropped the expression verbatim back in the output---so we didn't completely evaluate the SpEL.  If there's a better term for it, I can change the variable name.  (I took the term "partially evaluated" from the unit tests in the referenced task.)

Reply:
Ah got it, was caught up the canonical definition of [partial evaluation](https://en.wikipedia.org/wiki/Partial_evaluation). I'd probably pick something else, like `failedEvaluation`, `expressionUnresolved`, or something similar.